lib/commands.coffee

```diff
@@ -50,7 +57,7 @@ class Commands
   lint: ->
     try
       @linter.getActiveEditorLinter()?.lint(false)
-      @linter.views.render()
+      @linter.views.render(@linter.messages.publicMessages)
```

Comment:
Why does the `Commands` class have to reach through `@linter` to give it something it already has? Seems like a violation of the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter).